docs/UserGuide.md

```diff
@@ -183,30 +222,84 @@ Examples:
 `del a/` -         Deletes all addresses of the contact.
 
 `del e/ Official` -   Deletes the Official email of the contact.
-   
-#### 2.4 Viewing help: `help`
+
+<br>
+
+###2.4 Managing Meetings
+####2.4.1 Creating a meeting
+You can create a meeting with the following information:
+- Agenda
+- Meeting place
+- Meeting date and time: in DD-MM-YYYY HH:mm format
+- Attendees: in the form of index numbers of people on the displayed list
```

Comment:
I think we should use a similar format for the earlier section on adding contacts too for clarity

Reply:
I was thinking of doing that in another PR, so that we can get the meeting commands added asap.